docs/topic_guides/evaluation/edges.md

```diff
@@ -1,3 +1,50 @@
 # Edge Evaluation
 
-This page is under construction - check back soon!
\ No newline at end of file
+Once you have a trained model, you use it to generate edges (links) between entities (nodes). These edges will have a Match Weight and corresponding Probability.
+
+There are several strategies for checking whether the links in created in your pipeline perform as you want/expect.
+
+## Consider the Edge Metrics
+
+Edge Metrics measure for how links perform at an overall level.
+
+First, consider how you would like your model to perform. What is important for your use case? Do you want to ensure that you capture all possible matches? Or do you want to minimise the number of incorrectly predicted matches? Perhaps a combination of both?
+
+For a summary of all the edge metrics available in Splink, check out the [Edge Metrics guide](./edge_metrics.md).
+
+!!! note
+
+    To produce Edge Metrics you will require a "ground truth" to compare your linkage results against (which can be achieved by [Clerical Labelling](./labelling.md))
+
+
+## Spot Checking pairs of records
+
+Spot Checking real examples of record pairs is helpful for  confidence in linkage results. It is an effective way to build up an intuition for how the model works in practice and allows you to interrogate edge cases.
+
+Results of individual record pairs can be examined with the [Waterfall Chart](../../charts/waterfall_chart.ipynb).
+
+Choosing which the pairs of records to spot check can be done by either:
+
+- Looking at all combinations of comparison levels and choosing which to examine in the [Comparison Viewer Dashboard](../../charts/comparison_viewer_dashboard.ipynb).
+- Identifying and examining records which have been [incorrectly predicted by your Splink model](../../demos/examples/duckdb/accuracy_analysis_from_labels_column.ipynb).
+
+As you are checking real examples, you will often come across cases that have not been accounted for by your model which you believe signify a match (e.g. a fuzzy match for names). We recommend using this feedback loop to help iterate and improve the definition of your model.
+
+## Choosing a Threshold
+
+Threshold selection is a key decision point within a linkage pipeline. One of the major benefits of probabilistic linkage versus a deterministic (i.e. rules-based) approach is the ability to choose the amount of evidence required for two records to be considered a match (i.e. a threshold). 
+
+When you have decided on the metrics that are important for your use case, you can use the [confusion matrix](../../charts/confusion_matrix_from_labels_table.ipynb) and [accuracy chart](../../charts/accuracy_chart_from_labels_table.ipynb) to get a first estimate for what your threshold should be.
```

Comment:
Is it worthwhile mentioning that these require labelled data?

Reply:
Yep, good shout. Adding now